Description: Disable fd limit when running testsuite
 We've seen the testsuite hitting the fd limit sometimes on Debian
 alpha and s390 buildds and several of the Ubuntu buildds.  Looking at
 the output of lsof reveals that there aren't extra fds open (at least
 not in runtest).  Best current guess is a timing difference of some
 sort with when the databases for the previous testcase are closed.
 Since this umlimit is intended to catch newly introduced leaks before
 they get into a release, just disabling this in the Debian builds is
 an acceptable workaround.
Author: Olly Betts <olly@survex.com>
Index: xapian-core/tests/runtest.in
===================================================================
--- xapian-core/tests/runtest.in	(revision 13449)
+++ xapian-core/tests/runtest.in	(working copy)
@@ -35,7 +35,7 @@
 # If ulimit supports it, limit the number of open file descriptors so we catch
 # file descriptor leaks sooner.  Not all platforms support changing this limit
 # but Linux does at least.
-ulimit -n 64 2>/dev/null || true
+#ulimit -n 64 2>/dev/null || true
 
 case $1 in
   *test|*test@EXEEXT@) ;;
